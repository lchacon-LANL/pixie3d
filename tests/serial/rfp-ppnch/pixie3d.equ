r  Bt Bz Jt Jz Vr Ez Et rho
 -0.00390625 -0.00781226  1.00006103 -0.03124905  3.99975586  0.00003125  0.00400000  0.00000000  1.00003050
  0.00390625  0.00781226  0.99993897  0.03124905  3.99975586 -0.00003125  0.00400000  0.00000000  0.99996950
  0.01171875  0.02343106  0.99945076  0.09372424  3.99780273 -0.00009378  0.00400000  0.00000000  0.99972536
  0.01953125  0.03903270  0.99847470  0.15613070  3.99389649 -0.00015637  0.00400000  0.00000000  0.99923708
  0.02734375  0.05460572  0.99701152  0.21842240  3.98803714 -0.00021908  0.00400000  0.00000000  0.99850466
  0.03515625  0.07013869  0.99506227  0.28055306  3.98022473 -0.00028195  0.00400000  0.00000000  0.99752810
  0.04296875  0.08562017  0.99262842  0.34247599  3.97045939 -0.00034502  0.00400000  0.00000000  0.99630741
  0.05078125  0.10103870  0.98971175  0.40414402  3.95874133 -0.00040835  0.00400000  0.00000000  0.99484259
  0.05859375  0.11638286  0.98631445  0.46550938  3.94507095 -0.00047197  0.00400000  0.00000000  0.99313367
  0.06640625  0.13164120  0.98243908  0.52652358  3.92944886 -0.00053594  0.00400000  0.00000000  0.99118067
  0.07421875  0.14680229  0.97808856  0.58713736  3.91187597 -0.00060029  0.00400000  0.00000000  0.98898362
  0.08203125  0.16185471  0.97326623  0.64730053  3.89235357 -0.00066508  0.00400000  0.00000000  0.98654258
  0.08984375  0.17678704  0.96797580  0.70696192  3.87088344 -0.00073035  0.00400000  0.00000000  0.98385763
  0.09765625  0.19158789  0.96222139  0.76606930  3.84746795 -0.00079615  0.00400000  0.00000000  0.98092886
  0.10546875  0.20624587  0.95600755  0.82456928  3.82211018 -0.00086251  0.00400000  0.00000000  0.97775640
  0.11328125  0.22074964  0.94933922  0.88240726  3.79481404 -0.00092950  0.00400000  0.00000000  0.97434042
  0.12109375  0.23508785  0.94222181  0.93952734  3.76558443 -0.00099714  0.00400000  0.00000000  0.97068112
  0.12890625  0.24924922  0.93466113  0.99587229  3.73442739 -0.00106549  0.00400000  0.00000000  0.96677876
  0.13671875  0.26322251  0.92666347  1.05138352  3.70135025 -0.00113459  0.00400000  0.00000000  0.96263363
  0.14453125  0.27699653  0.91823559  1.10600102  3.66636183 -0.00120449  0.00400000  0.00000000  0.95824612
  0.15234375  0.29056015  0.90938469  1.15966334  3.62947259 -0.00127522  0.00400000  0.00000000  0.95361666
  0.16015625  0.30390233  0.90011850  1.21230764  3.59069488 -0.00134683  0.00400000  0.00000000  0.94874577
  0.16796875  0.31701211  0.89044520  1.26386964  3.55004308 -0.00141937  0.00400000  0.00000000  0.94363406
  0.17578125  0.32987866  0.88037352  1.31428368  3.50753388 -0.00149287  0.00400000  0.00000000  0.93828224
  0.18359375  0.34249123  0.86991269  1.36348281  3.46318646 -0.00156738  0.00400000  0.00000000  0.93269112
  0.19140625  0.35483925  0.85907245  1.41139877  3.41702276 -0.00164293  0.00400000  0.00000000  0.92686163
  0.19921875  0.36691229  0.84786310  1.45796219  3.36906766 -0.00171957  0.00400000  0.00000000  0.92079484
  0.20703125  0.37870012  0.83629549  1.50310263  3.31934927 -0.00179733  0.00400000  0.00000000  0.91449195
  0.21484375  0.39019268  0.82438102  1.54674876  3.26789915 -0.00187625  0.00400000  0.00000000  0.90795432
  0.22265625  0.40138016  0.81213162  1.58882855  3.21475258 -0.00195637  0.00400000  0.00000000  0.90118347
  0.23046875  0.41225302  0.79955984  1.62926943  3.15994875 -0.00203771  0.00400000  0.00000000  0.89418112
  0.23828125  0.42280196  0.78667875  1.66799856  3.10353105 -0.00212030  0.00400000  0.00000000  0.88694914
  0.24609375  0.43301804  0.77350201  1.70494308  3.04554727 -0.00220419  0.00400000  0.00000000  0.87948965
  0.25390625  0.44289261  0.76004385  1.74003040  2.98604979 -0.00228938  0.00400000  0.00000000  0.87180495
  0.26171875  0.45241745  0.74631906  1.77318855  2.92509586 -0.00237591  0.00400000  0.00000000  0.86389761
  0.26953125  0.46158470  0.73234298  1.80434653  2.86274769 -0.00246380  0.00400000  0.00000000  0.85577042
  0.27734375  0.47038696  0.71813153  1.83343472  2.79907267 -0.00255306  0.00400000  0.00000000  0.84742643
  0.28515625  0.47881731  0.70370113  1.86038527  2.73414348 -0.00264372  0.00400000  0.00000000  0.83886898
  0.29296875  0.48686934  0.68906875  1.88513262  2.66803817 -0.00273577  0.00400000  0.00000000  0.83010167
  0.30078125  0.49453718  0.67425184  1.90761395  2.60084026 -0.00282923  0.00400000  0.00000000  0.82112841
  0.30859375  0.50181557  0.65926834  1.92776970  2.53263870 -0.00292410  0.00400000  0.00000000  0.81195342
  0.31640625  0.50869983  0.64413664  1.94554411  2.46352791 -0.00302039  0.00400000  0.00000000  0.80258125
  0.32421875  0.51518599  0.62887555  1.96088578  2.39360762 -0.00311808  0.00400000  0.00000000  0.79301675
  0.33203125  0.52127073  0.61350425  1.97374824  2.32298280 -0.00321717  0.00400000  0.00000000  0.78326514
  0.33984375  0.52695147  0.59804229  1.98409052  2.25176340 -0.00331764  0.00400000  0.00000000  0.77333196
  0.34765625  0.53222640  0.58250949  1.99187773  2.18006414 -0.00341948  0.00400000  0.00000000  0.76322311
  0.35546875  0.53709448  0.56692593  1.99708165  2.10800412 -0.00352265  0.00400000  0.00000000  0.75294485
  0.36328125  0.54155548  0.55131188  1.99968123  2.03570650 -0.00362713  0.00400000  0.00000000  0.74250380
  0.37109375  0.54561003  0.53568773  1.99966321  1.96329794 -0.00373289  0.00400000  0.00000000  0.73190693
  0.37890625  0.54925959  0.52007397  1.99702252  1.89090813 -0.00383988  0.00400000  0.00000000  0.72116156
  0.38671875  0.55250652  0.50449107  1.99176282  1.81866916 -0.00394806  0.00400000  0.00000000  0.71027536
  0.39453125  0.55535403  0.48895946  1.98389683  1.74671486 -0.00405739  0.00400000  0.00000000  0.69925638
  0.40234375  0.55780624  0.47349942  1.97344673  1.67518004 -0.00416779  0.00400000  0.00000000  0.68811295
  0.41015625  0.55986815  0.45813101  1.96044438  1.60419976 -0.00427922  0.00400000  0.00000000  0.67685377
  0.41796875  0.56154566  0.44287401  1.94493153  1.53390846 -0.00439161  0.00400000  0.00000000  0.66548781
  0.42578125  0.56284551  0.42774785  1.92695992  1.46443911 -0.00450490  0.00400000  0.00000000  0.65402437
  0.43359375  0.56377531  0.41277151  1.90659123  1.39592231 -0.00461900  0.00400000  0.00000000  0.64247297
  0.44140625  0.56434351  0.39796342  1.88389706  1.32848541 -0.00473384  0.00400000  0.00000000  0.63084343
  0.44921875  0.56455934  0.38334146  1.85895865  1.26225158 -0.00484935  0.00400000  0.00000000  0.61914576
  0.45703125  0.56443280  0.36892280  1.83186658  1.19733891 -0.00496545  0.00400000  0.00000000  0.60739017
  0.46484375  0.56397460  0.35472389  1.80272036  1.13385955 -0.00508204  0.00400000  0.00000000  0.59558703
  0.47265625  0.56319613  0.34076037  1.77162792  1.07191891 -0.00519904  0.00400000  0.00000000  0.58374685
  0.48046875  0.56210938  0.32704698  1.73870491  1.01161483 -0.00531638  0.00400000  0.00000000  0.57188022
  0.48828125  0.56072690  0.31359755  1.70407404  0.95303693 -0.00543395  0.00400000  0.00000000  0.55999782
  0.49609375  0.55906174  0.30042493  1.66786424  0.89626595 -0.00555168  0.00400000  0.00000000  0.54811034
  0.50390625  0.55712737  0.28754093  1.63020980  0.84137323 -0.00566949  0.00400000  0.00000000  0.53622844
  0.51171875  0.55493759  0.27495630  1.59124938  0.78842028 -0.00578728  0.00400000  0.00000000  0.52436277
  0.51953125  0.55250651  0.26268069  1.55112503  0.73745846 -0.00590498  0.00400000  0.00000000  0.51252386
  0.52734375  0.54984843  0.25072263  1.50998118  0.68852875 -0.00602252  0.00400000  0.00000000  0.50072212
  0.53515625  0.54697779  0.23908951  1.46796357  0.64166169 -0.00613981  0.00400000  0.00000000  0.48896781
  0.54296875  0.54390908  0.22778760  1.42521821  0.59687739 -0.00625679  0.00400000  0.00000000  0.47727100
  0.55078125  0.54065676  0.21682200  1.38189033  0.55418566 -0.00637339  0.00400000  0.00000000  0.46564150
  0.55859375  0.53723522  0.20619670  1.33812339  0.51358627 -0.00648955  0.00400000  0.00000000  0.45408888
  0.56640625  0.53365869  0.19591459  1.29405812  0.47506932 -0.00660522  0.00400000  0.00000000  0.44262241
  0.57421875  0.52994118  0.18597746  1.24983158  0.43861567 -0.00672034  0.00400000  0.00000000  0.43125105
  0.58203125  0.52609642  0.17638605  1.20557636  0.40419749 -0.00683487  0.00400000  0.00000000  0.41998340
  0.58984375  0.52213778  0.16714008  1.16141980  0.37177888 -0.00694878  0.00400000  0.00000000  0.40882769
  0.59765625  0.51807826  0.15823829  1.11748335  0.34131650 -0.00706203  0.00400000  0.00000000  0.39779178
  0.60546875  0.51393043  0.14967853  1.07388192  0.31276036 -0.00717459  0.00400000  0.00000000  0.38688310
  0.61328125  0.50970637  0.14145773  1.03072346  0.28605451 -0.00728644  0.00400000  0.00000000  0.37610868
  0.62109375  0.50541765  0.13357206  0.98810854  0.26113787 -0.00739757  0.00400000  0.00000000  0.36547512
  0.62890625  0.50107532  0.12601688  0.94613007  0.23794498 -0.00750796  0.00400000  0.00000000  0.35498857
  0.63671875  0.49668986  0.11878689  0.90487307  0.21640680 -0.00761762  0.00400000  0.00000000  0.34465475
  0.64453125  0.49227118  0.11187616  0.86441461  0.19645145 -0.00772653  0.00400000  0.00000000  0.33447894
  0.65234375  0.48782858  0.10527816  0.82482372  0.17800499 -0.00783471  0.00400000  0.00000000  0.32446596
  0.66015625  0.48337080  0.09898587  0.78616153  0.16099210 -0.00794216  0.00400000  0.00000000  0.31462020
  0.66796875  0.47890596  0.09299182  0.74848131  0.14533676 -0.00804889  0.00400000  0.00000000  0.30494561
  0.67578125  0.47444162  0.08728817  0.71182872  0.13096285 -0.00815493  0.00400000  0.00000000  0.29544572
  0.68359375  0.46998473  0.08186672  0.67624202  0.11779472 -0.00826028  0.00400000  0.00000000  0.28612362
  0.69140625  0.46554169  0.07671903  0.64175239  0.10575771 -0.00836497  0.00400000  0.00000000  0.27698201
  0.69921875  0.46111834  0.07183642  0.60838424  0.09477859 -0.00846902  0.00400000  0.00000000  0.26802318
  0.70703125  0.45671997  0.06721006  0.57615560  0.08478598 -0.00857246  0.00400000  0.00000000  0.25924903
  0.71484375  0.45235137  0.06283099  0.54507846  0.07571066 -0.00867531  0.00400000  0.00000000  0.25066111
  0.72265625  0.44801684  0.05869019  0.51515924  0.06748585 -0.00877760  0.00400000  0.00000000  0.24226059
  0.73046875  0.44372018  0.05477861  0.48639915  0.06004746 -0.00887936  0.00400000  0.00000000  0.23404831
  0.73828125  0.43946479  0.05108720  0.45879463  0.05333426 -0.00898062  0.00400000  0.00000000  0.22602477
  0.74609375  0.43525361  0.04760695  0.43233777  0.04728803 -0.00908140  0.00400000  0.00000000  0.21819018
  0.75390625  0.43108920  0.04432896  0.40701673  0.04185358 -0.00918173  0.00400000  0.00000000  0.21054444
  0.76171875  0.42697375  0.04124440  0.38281610  0.03697890 -0.00928165  0.00400000  0.00000000  0.20308718
  0.76953125  0.42290912  0.03834459  0.35971734  0.03261508 -0.00938118  0.00400000  0.00000000  0.19581775
  0.77734375  0.41889683  0.03562100  0.33769914  0.02871633 -0.00948034  0.00400000  0.00000000  0.18873527
  0.78515625  0.41493814  0.03306529  0.31673777  0.02523997 -0.00957916  0.00400000  0.00000000  0.18183863
  0.79296875  0.41103402  0.03066929  0.29680742  0.02214628 -0.00967768  0.00400000  0.00000000  0.17512650
  0.80078125  0.40718519  0.02842506  0.27788051  0.01939847 -0.00977590  0.00400000  0.00000000  0.16859733
  0.80859375  0.40339218  0.02632487  0.25992803  0.01696258 -0.00987386  0.00400000  0.00000000  0.16224940
  0.81640625  0.39965529  0.02436122  0.24291977  0.01480732 -0.00997158  0.00400000  0.00000000  0.15608083
  0.82421875  0.39597465  0.02252687  0.22682459  0.01290398 -0.01006907  0.00400000  0.00000000  0.15008954
  0.83203125  0.39235025  0.02081479  0.21161069  0.01122628 -0.01016636  0.00400000  0.00000000  0.14427333
  0.83984375  0.38878190  0.01921824  0.19724576  0.00975024 -0.01026347  0.00400000  0.00000000  0.13862987
  0.84765625  0.38526933  0.01773070  0.18369725  0.00845404 -0.01036040  0.00400000  0.00000000  0.13315669
  0.85546875  0.38181211  0.01634593  0.17093252  0.00731787 -0.01045719  0.00400000  0.00000000  0.12785121
  0.86328125  0.37840975  0.01505793  0.15891896  0.00632381 -0.01055384  0.00400000  0.00000000  0.12271075
  0.87109375  0.37506167  0.01386095  0.14762420  0.00545567 -0.01065037  0.00400000  0.00000000  0.11773254
  0.87890625  0.37176720  0.01274951  0.13701621  0.00469888 -0.01074678  0.00400000  0.00000000  0.11291373
  0.88671875  0.36852564  0.01171837  0.12706338  0.00404036 -0.01084310  0.00400000  0.00000000  0.10825140
  0.89453125  0.36533623  0.01076252  0.11773469  0.00346837 -0.01093932  0.00400000  0.00000000  0.10374256
  0.90234375  0.36219814  0.00987721  0.10899973  0.00297244 -0.01103547  0.00400000  0.00000000  0.09938418
  0.91015625  0.35911056  0.00905793  0.10082879  0.00254323 -0.01113155  0.00400000  0.00000000  0.09517316
  0.91796875  0.35607260  0.00830037  0.09319295  0.00217241 -0.01122756  0.00400000  0.00000000  0.09110638
  0.92578125  0.35308339  0.00760047  0.08606408  0.00185261 -0.01132352  0.00400000  0.00000000  0.08718068
  0.93359375  0.35014203  0.00695437  0.07941496  0.00157731 -0.01141943  0.00400000  0.00000000  0.08339287
  0.94140625  0.34724762  0.00635843  0.07321921  0.00134071 -0.01151530  0.00400000  0.00000000  0.07973977
  0.94921875  0.34439923  0.00580921  0.06745142  0.00113775 -0.01161112  0.00400000  0.00000000  0.07621815
  0.95703125  0.34159597  0.00530345  0.06208707  0.00096393 -0.01170692  0.00400000  0.00000000  0.07282480
  0.96484375  0.33883692  0.00483811  0.05710264  0.00081534 -0.01180268  0.00400000  0.00000000  0.06955650
  0.97265625  0.33612117  0.00441029  0.05247552  0.00068854 -0.01189842  0.00400000  0.00000000  0.06641003
  0.98046875  0.33344784  0.00401730  0.04818408  0.00058051 -0.01199414  0.00400000  0.00000000  0.06338220
  0.98828125  0.33081604  0.00365660  0.04420764  0.00048864 -0.01208984  0.00400000  0.00000000  0.06046979
  0.99609375  0.32822489  0.00332579  0.04052646  0.00041064 -0.01218552  0.00400000  0.00000000  0.05766966
  1.00390625  0.32566370  0.00301600  0.03705488  0.00034179 -0.01228118  0.00400000 -0.00000000  0.05495175

eta
  0.00100000
delta
  0.90000000