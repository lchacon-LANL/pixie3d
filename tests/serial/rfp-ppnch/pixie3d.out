 Calculating equilibrium ppnch...
 
 Equilibrium converged in  10 iterations with error =  1.85E-06
 
 Force-free error = 4.3041172E-05
 Ohms law   error = 8.2479882E-05
 
 Pinch equilibrium features:
   Theta  = 1.6482468
   Bz(r=0)= 1.0000000
   I0     = 2.0544812
   E0/eta = 4.0000000
   T_flux = 0.6232323
 
 ***********************************************
 ****** PIXIE3D Extended MHD code, V. 0.9 ******
 ******       Developer: L. Chacon        ******
 ******  Los Alamos National Laboratory   ******
 ******       Los Alamos, NM 87545        ******
 ******           LA-CC 07-005            ******
 ***********************************************
 
 ****************** NOTICES ******************
 Using MAGNETIC FIELD version
 Using non-conservative JxB     part of EOM
 Using non-conservative grad(p) part of EOM
 Not solving for density: rho = 1
 ****************** NOTICES ******************

  Grid mesh:................  128x   32x    1
  Number of grid levels:....    6,    4,    1
  Preconditioning method:... id        
  Time correction...........   F
  External source...........   T
  Numerical grid............   F
  Advection scheme..........   ZIP    
  TS scheme................. Theta    
 
   itime    time     dt    vx_max vy_max vz_max GMRES  Newton  CN factor
      1    0.005  5.0E-03   0.012  0.000  0.000     3     3     0.500
      2    0.010  5.0E-03   0.012  0.000  0.000     3     3     0.500
      3    0.015  5.0E-03   0.012  0.000  0.000     4     3     0.500
      4    0.020  5.0E-03   0.012  0.000  0.000     7     5     0.500
      5    0.025  5.0E-03   0.012  0.000  0.000    10     5     0.500
      6    0.030  5.0E-03   0.012  0.000  0.000    10     5     0.500
      7    0.035  5.0E-03   0.012  0.000  0.000    15     5     0.500
      8    0.040  5.0E-03   0.012  0.000  0.000    11     5     0.500
      9    0.045  5.0E-03   0.012  0.000  0.000    14     5     0.500
     10    0.050  5.0E-03   0.012  0.000  0.000    14     5     0.500

Final statistics

 itime  Newt/ndt  GMRES/ndt  GMRES/Newt
   10       4.4        9.1        2.1
